quipucords/fingerprinter/jboss_brms.py

```diff
@@ -138,25 +127,42 @@ def detect_jboss_brms(source, facts):
     }
     product_dict = {'name': PRODUCT}
 
+    versions = set()
```

Comment:
Do we have issues if they have BRMS installed at the same version multiple times on the same system?

Reply:
I designed this to deduplicate versions on a single system, and it sounds like you want to keep the version information for each installation. Is that right? I will go ahead and work on that change.